Tests/base/NSProxy/basic.m

```diff
@@ -1,6 +1,9 @@
 #import "ObjectTesting.h"
 #import <Foundation/NSAutoreleasePool.h>
 #import <Foundation/NSProxy.h>
+#if __has_include(<objc/capabilities.h>
```

Comment:
Here a closing bracket ")" is missing. This doesn't look like you ever did run this patch. I rather withdraw my approval until I have time to actually test this change myself.

Reply:
Ooops.  I ran the tests and then the ones that failed test for functionality that's gone away from OS X since 10.7, so I tried to disable those and made a typo.  I've now pushed a fix and all of the tests are passing for me (except the `NSNumberFormatter` one that was failing before).